test/functional/apps/saved_objects_management/edit_saved_object.ts

```diff
@@ -35,6 +37,26 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
     return testSubjects.getAttribute(`savedObjects-editField-${fieldName}`, 'value');
   };
 
+  const setAceEditorFieldValue = async (fieldName: string, fieldValue: string) => {
+    const editorId = `savedObjects-editField-${fieldName}-aceEditor`;
+    await find.clickByCssSelector(`#${editorId}`);
+    return browser.execute(
+      (editor: string, value: string) => {
+        return (window as any).ace.edit(editor).setValue(value);
+      },
+      editorId,
```

Comment:
Using the `ace` API from the browser side instead of performing 'real' user interactions  is far from ideal, however as `ace` autocompletes user input (for example,  user type `[`, ace automatically adds the closing `]`), it was just over complicated to use `focus` + `sendKeys`, so this will have to be good enough.

Same things with `getAceEditorFieldValue`: parsing the text content from the ace editor structure is a pain.